README.md

```diff
@@ -79,9 +79,9 @@ A [Workflow Management System](https://en.wikipedia.org/wiki/Workflow_management
 
 The [WDL website](https://software.broadinstitute.org/wdl/) is the best place to go for more information on both WDL and Cromwell. In particular new users should check out the [user guide](https://software.broadinstitute.org/wdl/userguide/) which has many tutorials, examples and other bits to get you started.
 
-## User Forum
+## Support Forum
 
-If you have questions that aren't covered by the website you can ask them in the [user forum](http://gatkforums.broadinstitute.org/wdl/entry/signin?Target=categories%2Fask-the-wdl-team).
+If you have questions that aren't covered by the website you can ask them in the [support forum](http://gatkforums.broadinstitute.org/wdl/entry/signin?Target=categories%2Fask-the-wdl-team).
```

Comment:
Are you using the sign-in entry point on purpose? I would prefer to link directly to the category as some people (anyone coming in from GATK) will already be signed in.